catt/controllers.py

```diff
@@ -35,8 +41,35 @@ def get_chromecast(device_name):
         return devices[0]
 
 
-def human_time(seconds):
-    return time.strftime("%H:%M:%S", time.gmtime(seconds))
+def setup_cast(device_name, video_url=None, prep=None):
```

Comment:
Can you add a docstring explaining what this function does and the arguments?

Reply:
Will do.

Reply:
Still no docstring though :P

Reply:
I edited the todo list :P